tests/aftu/test_compare_graphs.py

```diff
@@ -0,0 +1,174 @@
+"""Compare graphs generated by vLLM-Spyre vs AFTU
+
+Run `python -m pytest tests/aftu/test_compare_graphs.py`.
+"""
+
+import os
+import sys
+import tempfile
+
+import pytest
+from graph_compare_utils import (collect_graph_files, compare_graphs,
+                                 get_model_path,
+                                 run_inference_py_and_get_graphs)
+from spyre_util import (generate_spyre_vllm_output, get_chicken_soup_prompts,
+                        get_spyre_model_list)
+from vllm import SamplingParams
+
+
+@pytest.mark.spyre
+@pytest.mark.aftu
+@pytest.mark.parametrize("model", get_spyre_model_list())
+@pytest.mark.parametrize("max_num_seqs", [2, 4],
```

Comment:
I wonder if we can have only 4 here to speed things up unless I'm missing something?

Reply:
Probably? We initially had 2/4 on all the CB tests because batch size 4 didn't work